content/aips/8f46ded2d187-AIP-CHAOS-LABS-RISK-PLATFORM-FOR-V3.md

```diff
@@ -0,0 +1,52 @@
+| Title                             | Status | Author        | Short Description                                      | discussions                                                                                                                                                                          | created     |
```

Comment:
Could you please use the structure that is detailed here?
https://github.com/aave/aip/blob/master/X-AIP.md?plain=1